package/MDAnalysis/coordinates/TXYZ.py

```diff
@@ -35,6 +35,10 @@
 .. _format: http://chembytes.wikidot.com/tnk-tut00#toc2
 .. _Tinker: https://dasher.wustl.edu/tinker/
 
+
+Classes
+-------
```

Comment:
You either need to add `:inherited-members:` to the coordinates/TKYX.rst file or explicitly list the class here with members and inherited-members (and remove `:members:` from the rst).